pyproject.toml

```diff
@@ -56,3 +56,43 @@ line-length = 88
 [tool.isort]
 profile = "black"
 multi_line_output = 3
+
+[tool.pixi.project]
+channels = ["conda-forge"]
+platforms = ["linux-64", "linux-aarch64", "win-64", "osx-arm64"]
```

Comment:
Does `win-64` work locally? If we get the same failure of the tests, it can be removed

Reply:
I prefer to leave as I plan to fix the tests during next week, and apart from the few tests the library itself works fine.

Reply:
Great, thanks for the explanation